python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -57,48 +57,20 @@ def validate_environment() -> None:
 def parse_args(args: List[str]) -> argparse.Namespace:
     """parse args for running this entrypoint script"""
     parser = argparse.ArgumentParser(description=DESCRIPTION)
-    parser.add_argument(
-        "--experimental",
-        action="store_true",
-        dest="experimental",
-        help="if set, use a sqllite engine for quicker development",
-    )
-
```

Comment:
remove these from the performance manager service. we don't want that service to be in charge of seeding the metadata table so that we can do it locally for testing. this functionality is now in a script we can run.